lib/vizkit/cplusplus_extensions/vizkit_widget.rb

```diff
@@ -324,7 +324,7 @@ def push_transformer_configuration(data)
     def listen_to_transformation_producer(trsf)
         return if @connected_transformation_producers.has_key?(trsf.producer)
 
-        task, *port = trsf.producer.split('.')
+        task, bla, *port = trsf.producer.rpartition('.')
```

Comment:
Maybe we should:
 * rename bla to sep for separator
 * change *port to port 
 * remove line 328. 

Other than that the fix solves the problem :+1: